crates/sui-config/src/genesis.rs

```diff
@@ -55,10 +56,9 @@ use tracing::trace;
 pub struct Genesis {
     checkpoint: CertifiedCheckpointSummary,
     checkpoint_contents: CheckpointContents,
-    transaction: CertifiedTransaction,
+    transaction: Transaction,
```

Comment:
Wondering what's the rationale for making this not certified anymore? Is it so that each validator can sign this themselves and process the genesis txn like a regular txn? Apologies if my guess is very off.

Reply:
This only needed to be certified before @lxfind 's Finalized transaction work, since the checkpoint itself is certified this isn't needed anymore